spyder/api/panel.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Copyright © <2013-2016> <Colin Duquesnoy and others, see pyqode/AUTHORS.rst>
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+This module contains the panel API.
+Adapted from https://github.com/pyQode/pyqode.core/blob/master/pyqode/core/api/panel.py
+"""
+import logging
+
+from qtpy.QtWidgets import QWidget, QApplication
+from qtpy.QtGui import QBrush, QColor, QPen, QPainter
+
+from spyder.api.mode import Mode
+
+def _logger():
+    """ Returns module's logger """
+    return logging.getLogger(__name__)
+
+
+class Panel(QWidget, Mode):
+    """
+    Base class for editor panels.
+
+    A panel is a mode and a QWidget.
+
+    .. note:: Use enabled to disable panel actions and setVisible to change the
+        visibility of the panel.
+    """
+    class Position(object):
+        """
+        Enumerates the possible panel positions
+        """
+        #: Top margin
+        TOP = 0
+        #: Left margin
+        LEFT = 1
+        #: Right margin
+        RIGHT = 2
+        #: Bottom margin
+        BOTTOM = 3
+
+        @classmethod
+        def iterable(cls):
+            """ Returns possible positions as an iterable (list) """
+            return [cls.TOP, cls.LEFT, cls.RIGHT, cls.BOTTOM]
+
+    @property
+    def scrollable(self):
+        """
+        A scrollable panel will follow the editor's scroll-bars. Left and right
```

Comment:
Change to

    A scrollable panel will follow the editor's scroll-bars.

    Left and right ...